### ISSUE TITLE
Type ahead matching field type - screen readers only say "edit text" when tabbing into fields that have no associated label 

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2827]
There is a type-ahead item-matching field type found on several pages (Advanced search, Numismatic Search, etc) wherein the fields are not associated with a label, so screen readers only say "edit text" when tabbing into them.

Use the label's for= attribute or the input's aria-labelledby attribute to cross-reference the label.

https://www.w3.org/WAI/WCAG21/quickref/#label-in-name

**Comment by rladdusaw** [comment]
This issue seems to have been resolved already. I was unable to find any fields that did not already have an associated label for the text field.
